packages/editor-ui/src/api/users.ts

```diff
@@ -0,0 +1,71 @@
+import { IPersonalizationSurveyAnswers, IRestApiContext, IUserResponse } from '@/Interface';
+import { IDataObject } from 'n8n-workflow';
+import { makeRestApiRequest } from './helpers';
+
+export async function loginCurrentUser(context: IRestApiContext): Promise<IUserResponse | null> {
+	return await makeRestApiRequest(context, 'GET', '/login');
+}
+
+export async function getCurrentUser(context: IRestApiContext): Promise<IUserResponse | null> {
+	return await makeRestApiRequest(context, 'GET', '/me');
+}
+
+export async function login(context: IRestApiContext, params: {email: string, password: string}): Promise<IUserResponse> {
+	return await makeRestApiRequest(context, 'POST', '/login', params);
+}
+
+export async function logout(context: IRestApiContext): Promise<void> {
+	await makeRestApiRequest(context, 'POST', '/logout');
+}
+
+export async function setupOwner(context: IRestApiContext, params: { firstName: string; lastName: string; email: string; password: string;}): Promise<IUserResponse> {
+	return await makeRestApiRequest(context, 'POST', '/owner-setup', params as unknown as IDataObject);
```

Comment:
I think this changed
```suggestion
	return await makeRestApiRequest(context, 'POST', '/owner/setup', params as unknown as IDataObject);
```